5001-6000/LIT5407BigFeastsTig.xml

```diff
@@ -40,8 +40,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
             <textClass>
                 <keywords>
-                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
                     <term key="TigrinyaLiterature"/>
+                    <term key="Other"/>
+                    <!--Chronography?-->
```

Comment:
Well, I see the reason, but chronography does not really fits here. We do not have CalendarandComputus, do we?